scripts/microbe_stage/microbe_replacement.lua

```diff
@@ -64,5 +54,20 @@ function MicrobeReplacementSystem:activate()
     end
 end
 
+--Faux-latin name generation routine (Move to own file eventually?)
+function MicrobeReplacementSystem:generateSpeciesName()
+
+    --prefix,cofix,suffix list
+    speciesNamePrefix = {' Ce', ' Ar',' Sp', ' Th',' Co', ' So', ' Pu', ' Cr', ' Cy', ' Gr', ' Re', ' Ty', ' Tr', ' Ac',' Pr' }
+    speciesNameCofix = { 'nan', 'mo', 'na', 'yt', 'yn', 'il', 'li','le', 'op', 'un', 'rive','ec', 'ro','lar','im' }
+    speciesNameSuffix = { 'pien', 'olera', 'rius', 'nien', 'ster', 'ilia', 'canus', 'tus', 'cys','ium','um'}
+
+    --Generate random seed
+    math.randomseed(os.time())    
+    speciesGenName = (speciesNamePrefix[math.random(#speciesNamePrefix)]) .. (speciesNameCofix[math.random(#speciesNameCofix)]) .. (speciesNameSuffix[math.random(#speciesNameSuffix)])
+    local new_species_name = global_Genus_Name .. speciesGenName
+    return new_species_name;   
```

Comment:
No need for a local variable for new_species_name if you are returning it right away.